server/src/main/java/fr/uga/l3miage/tp1/exo2/models/User.java

```diff
@@ -0,0 +1,20 @@
+package fr.uga.l3miage.tp1.exo2.models;
+
+import javax.persistence.*;
+import java.time.OffsetDateTime;
+
+@Entity
+@Table(name = "users")
+public class User {
```

Comment:
Toutes tes entités doivent finir avec le suffixe `Entity` donc `UserEntity`
```suggestion
public class UserEntity {
```